package/CHANGELOG

```diff
@@ -52,6 +52,7 @@ Fixes
  * Fix groups.py doctests using sphinx directives (Issue #3925, PR #4374)
 
 Enhancements
+ * Improve error message for `AtomGroup.unwrap()` when bonds are not present.(Issue #4436, PR #4642)
```

Comment:
This _looks_ as if the line is too long an should be broken.

Please check that this is within the line length (see top of file) and add a line break if necessary.

If this is within line length just add a comment saying so --- when I am looking at it inside the GitHub interface it looks too long but I may be wrong.